crates/rattler_solve/src/lib.rs

```diff
@@ -46,6 +48,7 @@ impl fmt::Display for SolveError {
 
 /// Represents a dependency resolution task, to be solved by one of the backends (currently only
 /// libsolv is supported)
+#[cfg_attr(test, derive(Clone))]
```

Comment:
This seems like a hack. Why should it only be cloneable for tests?

Reply:
Oh, it's just that it's not "necessary" for it to be cloneable in the normal case, so I thought I'd be smart and only generate the clone impl for the case in which we are actually using it (when contributing to other Rust crates, sometimes maintainers prefer to avoid adding `Clone` impls for types in the public API unless it's truly necessary).

If you prefer I can change it to `derive(Clone)`